yarn.lock

```diff
@@ -21,7 +14,7 @@
   resolved "https://registry.npmjs.org/@babel/compat-data/-/compat-data-7.17.7.tgz#078d8b833fbbcc95286613be8c716cef2b519fa2"
   integrity sha512-p8pdE6j0a29TNGebNm7NzYZWB3xVZJBZ7XGs42uAKzQo8VQ3F0By/cQCtUEABwIqw5zo6WA4NbmxsfzADzMKnQ==
 
-"@babel/core@7.16.12":
+"@babel/core@7.16.12", "@babel/core@^7.1.0", "@babel/core@^7.10.2", "@babel/core@^7.7.5":
```

Comment:
Is the `yarn.lock` update related to the change in this PR?

Reply:
No it is a clean up that was applied by yarn. I can definitely drop it from this PR.

Reply:
Thanks. Maybe we can do the cleanup right after in a separate PR.